### PR TITLE
[Doc] Update FAQ for TensorRT

### PR DESCRIPTION
**Pull request by grimoire**
## Motivation

This PR add FAQ about TensorRT. And add assert about TensorRT input shapes.

## Modification

Please briefly describe what modification is made in this PR.

### CODE REVIEW

File: docs/en/faq.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+                input=dict(
+                    min_shape=[1, 3, 320, 320],
+                    opt_shape=[1, 3, 800, 1344],
+                    max_shape=[1, 3, 1344, 1344])))
+    ])
+  ```
+
+  The input tensor shape should be limited between `min_shape` and `max_shape`.
+
+- error: [TensorRT] INTERNAL ERROR: Assertion failed: cublasStatus == CUBLAS_STATUS_SUCCESS
```
Comment (AllentDan) [inline docs/en/faq.md:28]: May put ""  to or remove ""  from all error logs

Comment (grimoire) [reply docs/en/faq.md:28]: [MMDetection](https://github.com/open-mmlab/mmdetection/blob/master/docs/en/faq.md) also use "" in FAQ.

Comment (AllentDan) [reply docs/en/faq.md:28]: But the last item lack a `""`

**Comment by RunningLeon** [comment]
@VVsssssk Any  comment on this PR?

**Review by AllentDan — APPROVED** [review]

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**